django-mixin/dashboards/django-overview.json

```diff
@@ -0,0 +1,1262 @@
+          },
+          "custom": {
+            "axisLabel": "",
+            "axisPlacement": "left",
+            "axisSoftMax": 1,
+            "axisSoftMin": 0,
+            "barAlignment": 0,
+            "drawStyle": "line",
+            "fillOpacity": 0,
+            "gradientMode": "none",
+            "hideFrom": {
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "auto",
+            "spanNulls": false,
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "percentunit"
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Error rate"
+            },
+            "properties": [
+              {
+                "id": "color",
+                "value": {
+                  "fixedColor": "red",
+                  "mode": "fixed"
+                }
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 8,
+        "x": 16,
+        "y": 20
+      },
+      "id": 22,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom"
+        },
+        "tooltip": {
+          "mode": "single"
+        }
+      },
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "(sum(\n    rate(django_http_responses_total_by_status_view_method_total{job=~\"$job\", instance=~\"$instance\", status=~\"5..\", namespace=~\"$namespace\", app=~\"^$app\",view!~\"prometheus-django-metrics|healthcheck\"}[$__rate_interval])\n    ) or vector(0)) /\nsum(\n    rate(django_http_responses_total_by_status_view_method_total{job=~\"$job\", instance=~\"$instance\", namespace=~\"$namespace\", app=~\"^$app\",view!~\"prometheus-django-metrics|healthcheck\"}[$__rate_interval])\n    )",
+          "interval": "",
+          "legendFormat": "Error rate",
+          "refId": "A"
+        }
+      ],
+      "type": "timeseries"
+    }
+  ],
+  "refresh": false,
+  "schemaVersion": 34,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "current": {
+          "selected": false,
+          "text": "Prometheus",
+          "value": "Prometheus"
+        },
```

Comment:
@rgeyer any suggestions on how to remove this hard-coded values that come from my dev environment? Maybe just removing the whole `current` field works?

Reply:
Ahh, this is fine. The template will try to select the value from the dropdown which matches "current", but if it fails to do so it will simply revert to the next available prometheus datasource.

I was referring to the datasource for each target having been hard-coded to your dev datasource, rather than `$datasource` which you've already changed.

Thanks for sorting these out. This looks good to me!

Reply:
Gotcha. Thanks to you!